src/kibali/AuthZChecker.cs

```diff
@@ -31,7 +35,7 @@ public void Load(PermissionsDocument permissionsDocument)
 
         public ProtectedResource FindResource(string url)
         {
-            var parsedUrl = new Uri(new Uri("https://example.org/"), url, true);
+            var parsedUrl = new Uri(new Uri("https://example.org/"), url);
```

Comment:
Removing the don't escape means that we will need to escape the path in the permissions document which will mean all the { } are going to turn into percent encoded values.  I suspect this means we will have to decode before serializing the permissions document.

Reply:
Yeah this was flagged by CodeQL since the constructor is deprecated. We can leave it in but I don't know how long it will be supported.